ci/playbooks/set_crc_insecure_registry.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Patch the image.config.openshift.io resource to include insecure registry
+  ansible.builtin.shell: |
+    oc patch --type=merge --patch='{
+    "spec": {
+      "registrySources": {
+        "insecureRegistries": [
+        "{{ content_provider_registry_ip }}:5001"
+        ]
+      }
+    }
+    }' image.config.openshift.io/cluster
+
+- name: Patch the image.config.openshift.io resource to allow insecure registry
+  ansible.builtin.shell: |
+    oc patch --type=merge --patch='{
+    "spec": {
+      "registrySources": {
+        "allowedRegistries": [
+        "{{ content_provider_registry_ip }}:5001",
+        "quay.io",
+        "registry.redhat.io",
+        "image-registry.openshift-image-registry.svc:5000"
+        ]
+      }
+    }
+    }' image.config.openshift.io/cluster
+
+- name: Set Insecure registry for content provider
+  become: true
+  ansible.builtin.blockinfile:
+    state: present
+    insertafter: EOF
+    dest: /etc/containers/registries.conf
+    content: |-
+      [[registry]]
+      location = "{{ content_provider_registry_ip }}:5001"
+      insecure = true
+      blocked = false
+      mirror-by-digest-only = false
+      prefix = ""
+
+- name: Restart crio
+  become: true
+  ansible.builtin.service:
+    name: crio
+    state: restarted
+    enabled: true
```

Comment:
We shouldn't need this, it should be done by crc itself with the past patches.

Reply:
@pablintino Currently the podified deployment is failing while installing openstack operator with following [error](https://logserver.rdoproject.org/68/368/018327cc9f85736e69a4f7e90a4578b1685cc6c7/github-check/podified-multinode-edpm-deployment-crc/3713083/controller/ci-framework-data/logs/quay-io-openstack-k8s-operators-openstack-must-gather-sha256-070a98749455b097c66a1a25ff827a3c0ee7385f1f0c8bf54f5b15438501f2bd/namespaces/openstack-operators/all_resources.log)
```
NAME                                 READY   STATUS             RESTARTS   AGE
pod/openstack-operator-index-k8swn   0/1     ImagePullBackOff   0          6m3s

NAME                               TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)     AGE
service/openstack-operator-index   ClusterIP   10.217.4.74   <none>        50051/TCP   6m2s
```
In order to fix that, I need to add the content provider to `/etc/containers/registries.conf` otherwise it will not work, I have debugged the same last time also.

Reply:
Fair enough, if it doesn't work that's the way to for me is fine.